### DISCUSSION TITLE
Removable if not last child

### DISCUSSION THREAD

**Discussion by acxvgalanov** [GrapesJS/grapesjs#5513] [Q&A]
Can I make a component removable only if it has siblings?

E.g. I have a container Row and it's children of type Column. Can I allow Columns to be removed unless this is the last Column left in a Row? So the Row always has at least one Column in it?

**Comment by artf** [comment]
You can abort the remove
```js
editor.on('component:remove:before', (cmpToRemove, rm, opts) => {
    if (cmpToRemove.parent().components().length < 2) {
        opts.abort = true;
        alert('Last component inside')
    }
});
```
